### ISSUE TITLE
Notification widget becomes unstable when close to other users' Airpods

### ISSUE THREAD

**Issue by altShiftDev** [adolfintel/OpenPods#62]
I noticed this on the metro last week, the notification widget appears to be highly unstable, swapping between my own Airpod pros and regular Airpod icons while also showing various battery levels. 

My own were at 25% and every 3-5 seconds the notification widget would report them as normal Airpods at 95% or some other device around me - [screenshots.](https://imgur.com/a/Dy56Fxw)

I think what was happening was that it was reading battery and device information from other people around me (the behavior stopped as soon as I left the metro car). I find this odd though, I thought Bluetooth data like this was encrypted and only available to paired devices? How is it picking up this meta data from unpaired devices? It seems if nothing else the increased data processing from reading meta data from other people with Airpods is likely to drain your battery life much faster.

**Comment by adolfintel** [comment]
Yes this is a known issue, it's because android doesn't allow access to hardware addesses so the app doesn't know exactly which airpods are yours, it just picks up the strongest signal.
AirPods communicate their status over unencrypted BLE, not over the main bluetooth connection.
See https://github.com/adolfintel/OpenPods/blob/9a25fae80873159f45d6afe144536e50313ac322/OpenPods/app/src/main/java/com/dosse/airpods/PodsService.java#L60
